tests/e2e/enform.spec.js

```diff
@@ -0,0 +1,293 @@
+    await source.dragTo(target);
+
+    await page.locator('#publish').click();
+
+    // Assert that the fields were saved with the expected attributes/values.
+    for (const field of fields) {
+      const element = await page.locator(`tr[data-en-id="${field.id}"]`);
+      expect(element).toHaveAttribute('data-en-name', field.name);
+      if (formFieldsAttributes[field.name]?.required) {
+        await page.locator(`tr[data-en-id="${field.id}"] input[type="checkbox"]`).isChecked();
+      }
+      if (formFieldsAttributes[field.name]?.label?.length > 0 && field.type !== 'OPT') {
+        const labelField = await page.locator(`tr[data-en-id="${field.id}"] input[type="text"]`);
+        expect(labelField).toHaveValue(formFieldsAttributes[field.name].label);
+      }
+      if (formFieldsAttributes[field.name]?.default_value?.length > 0) {
+        await page.locator(`tr[data-en-id="${field.id}"] .dashicons-edit`).click();
+        const defaultValField = await page.locator(`.dialog-${field.id} input[data-attribute="default_value"]`)
+        await expect(defaultValField).toHaveValue(formFieldsAttributes[field.name].default_value);
+        await page.locator(`.dialog-${field.id} button[title="Close"]`).click();
+      }
+    }
+
+    const currentUrl = await page.url();
+    enFormId = new URL(currentUrl).searchParams.get('post');
+    console.log(`form ID: ${enFormId}`);
+  });
+
+  test('create Page with EN Form side style', async ({page, context}) => {
+    pageUrl = await createPageWithENForm(page, context, 'side');
+    console.log(`page URL: ${pageUrl}`);
+  });
+
+  test('fill EN Form (side style) on frontend', async ({page, context}) => {
+    await fillENFormAndSubmit(page, context);
+  });
+
+  test('create Page with EN Form full-width-bg style', async ({page, context}) => {
+    pageUrl = await createPageWithENForm(page, context, 'full-width-bg');
+    console.log(`page URL: ${pageUrl}`);
+  });
+
+  test('fill EN Form (full-width-bg style) on frontend', async ({page, context}) => {
+    await fillENFormAndSubmit(page, context);
+  });
+
+  test('disable EN Form block', async ({page, context}) => {
+
+    if (featureIsActiveOnInstance) {
+      console.log('Feature kept active.')
+      return;
+    }
+
+    await page.goto('./wp-admin/');
+    await login(context);
+
+    await page.goto('./wp-admin/admin.php?page=planet4_settings_features');
+    const checkbox = await page.getByRole('checkbox', {name: ' Activate the EN Form block, as well as the "Progress Bar inside EN Form" Counter block style.'});
+    const checked = await checkbox.isChecked();
+    if (checked) {
+      await checkbox.click();
+      await page.locator('input[type="submit"]').click();
+    }
+  });
+});
+
+async function createPageWithENForm(page, context, style) {
+  await page.goto('./wp-admin/');
+  await login(context);
+  await page.goto('./wp-admin/post-new.php?post_type=page');
```

Comment:
I've created a [function](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/main/tests/e2e/tools/lib/new-page.js#L4) to create a new page, maybe you could use it here?